python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -347,26 +347,45 @@ async def _tokenize_one_request(
                 f"model's context length ({self.context_len} tokens)."
             )
 
+        max_new_tokens_init = obj.sampling_params.get("max_new_tokens")
         if (
             obj.sampling_params.get("max_new_tokens") is not None
```

Comment:
Should we use
```suggestion
            max_new_tokens_init is not None
```
as well as the other places.

Reply:
Sure!